tensorflow_recommenders_addons/dynamic_embedding/python/ops/tf_patch.py

```diff
@@ -66,18 +69,21 @@ def update_op(self, optimizer, g):
           _apply_op = optimizer._resource_apply_sparse_duplicate_indices(
               g.values, self._v, g.indices)
         with ops.control_dependencies([_apply_op]):
-          _after = control_flow_ops.group([self._v.update_op()] +
-                                          [_s.update_op() for _s in _slots])
+          _after = control_flow_ops.group(
```

Comment:
Need to also apply the accumulative update [here](https://github.com/tensorflow/recommenders-addons/blob/master/tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_optimizer.py#L109-L124)

Reply:
Done. I'll test it later.